f8a_worker/storages/s3_kronos_append.py

```diff
@@ -0,0 +1,11 @@
+from . import AmazonS3
+
+
+class S3KronosAppend(AmazonS3):
+
+    def store_updated_data(self, input_json, file_name):
+        self.store_dict(input_json, file_name)
```

Comment:
Here you're just passing args from one method to another. The same in `fetch_existing_data`. Not really a problem though.

Reply:
Can we directly do that without creating a new storage class?

Reply:
Yes, I'd say it should be possible to use `AmazonS3` and its `store_dict()` and `retrieve_dict()` directly.

The `node.yaml` entry should I think (per [doc](http://selinon.readthedocs.io/en/latest/yaml.html#storages)) look like:
```
- name: 'S3KronosAppend'
  import: 'f8a_worker.storages'
  classname: 'AmazonS3'
```